sdk/netapp/Azure.ResourceManager.NetApp/src/Custom/NetAppAccountResource.cs

```diff
@@ -0,0 +1,92 @@
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Text;
+using System.Threading;
+using Azure.Core;
+using Azure.Core.Pipeline;
+using Azure.ResourceManager.NetApp;
+using Azure.ResourceManager.NetApp.Models;
+
+namespace Azure.ResourceManager.NetApp
+{
+    public partial class NetAppAccountResource: ArmResource
+    {
+        private VaultsRestOperations _vaultsRestClient;
+        private ClientDiagnostics _vaultsClientDiagnostics;
+
+        private ClientDiagnostics VaultsClientDiagnostics => _vaultsClientDiagnostics ??= new ClientDiagnostics("Azure.ResourceManager.NetApp", NetAppAccountResource.ResourceType.Namespace, Diagnostics);
+        private VaultsRestOperations VaultsRestClient => _vaultsRestClient ??= new VaultsRestOperations(Pipeline, Diagnostics.ApplicationId, Endpoint, GetApiVersionOrNull(NetAppAccountResource.ResourceType));
+
+        //        //
+        //        /// <summary> Initializes a new instance of the <see cref="NetAppAccountResource"/> class. </summary>
+        //        /// <param name="client"> The client parameters to use in these operations. </param>
+        //        /// <param name="id"> The identifier of the resource that is the target of operations. </param>
+        //        internal NetAppAccountResource(ArmClient client, ResourceIdentifier id) : base(client, id)
+        //        {
+        //            _netAppAccountAccountsClientDiagnostics = new ClientDiagnostics("Azure.ResourceManager.NetApp", ResourceType.Namespace, Diagnostics);
+        //            TryGetApiVersion(ResourceType, out string netAppAccountAccountsApiVersion);
+        //            _netAppAccountAccountsRestClient = new AccountsRestOperations(Pipeline, Diagnostics.ApplicationId, Endpoint, netAppAccountAccountsApiVersion);
+        //            _vaultsClientDiagnostics = new ClientDiagnostics("Azure.ResourceManager.NetApp", ProviderConstants.DefaultProviderNamespace, Diagnostics);
+        //            _vaultsRestClient = new VaultsRestOperations(Pipeline, Diagnostics.ApplicationId, Endpoint);
+        //            _netAppVolumeGroupVolumeGroupsClientDiagnostics = new ClientDiagnostics("Azure.ResourceManager.NetApp", NetAppVolumeGroupResource.ResourceType.Namespace, Diagnostics);
+        //            TryGetApiVersion(NetAppVolumeGroupResource.ResourceType, out string netAppVolumeGroupVolumeGroupsApiVersion);
+        //            _netAppVolumeGroupVolumeGroupsRestClient = new VolumeGroupsRestOperations(Pipeline, Diagnostics.ApplicationId, Endpoint, netAppVolumeGroupVolumeGroupsApiVersion);
+        //#if DEBUG
+        //            ValidateResourceId(Id);
+        //#endif
+        //        }
+        private string GetApiVersionOrNull(ResourceType resourceType)
+        {
+            TryGetApiVersion(resourceType, out string apiVersion);
+            return apiVersion;
+        }
```

Comment:
since we already have this method on the generated file and you did not change anything on this method, we could just remove this here, and regenerate to let the generated code handle this.